programmers/2일차/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+    let answer = "";
+    let splitS = s.split("");
+    if (splitS.length % 2 !== 0 ) {
+            answer = splitS[parseInt(splitS.length / 2)];
+            console.log(answer);
+    }
+    else {
+        answer = splitS[parseInt(splitS.length / 2)-1]+splitS[parseInt(splitS.length / 2)];
```

Comment:
```suggestion
        answer = splitS[parseInt(splitS.length / 2) - 1] + splitS[parseInt(splitS.length / 2)];
```
연산자 앞뒤로는  공백을 넣어주셔야 해요 ^^

```suggestion
        return splitS[parseInt(splitS.length / 2)-1]+splitS[parseInt(splitS.length / 2)];
```
이것 또한 그냥 return 해주셔도 될거같아요